packages/drivers/odsp-driver/src/prefetchLatestSnapshot.ts

```diff
@@ -82,7 +83,7 @@ export async function prefetchLatestSnapshot(
 		driveId: odspResolvedUrl.driveId,
 		itemId: odspResolvedUrl.itemId,
 	};
-	const storageTokenFetcher = toInstrumentedOdspTokenFetcher(
+	const tokenFetcher = toInstrumentedOdspTokenFetcher(
```

Comment:
In other places you named the var getAuthHeader.

Reply:
Should be fixed now